.travis.yml

```diff
@@ -35,7 +35,6 @@ addons:
     # Define the required packages without libjansson so we can do a
```

Comment:
Merge this with `github-ci: don't install libnss/libnspr packages`?

Reply:
We've typically never combined github-ci and travis-ci changes into the same commit.